torch/csrc/jit/export.cpp

```diff
@@ -653,6 +661,49 @@ class ScriptModuleSerializer {
           /*compress=*/true);
     }
   }
+
+  void writeByteCode(const script::Module& module) {
+    auto methods = module.get_methods();
+    std::vector<c10::IValue> elements;
+    for (const auto& method : methods) {
+      const auto& func = method.function();
+      torch::jit::Code code(func.graph());
+
+      // instructions
+      std::vector<IValue> inss;
+      for (const auto& ins : code.instructions()) {
+        TORCH_CHECK(isOpSupportedInMobile(ins.op), OpCode2Str(ins.op),
+                    " is not supported in mobile module.");
+        std::vector<IValue> insv{OpCode2Str(ins.op), ins.X, ins.N};
+        inss.emplace_back(c10::ivalue::Tuple::create(std::move(insv)));
+      }
+      auto instructions = c10::ivalue::Tuple::create(std::move(inss));
+      auto named_ins = c10::ivalue::Tuple::create({"instructions", instructions});
+
+      // operators
+      std::vector<IValue> opss;
+      for (const auto& opname : code.opname_table()) {
+        // Add "_" prefix to work around the double registration both of jit/generated
+        // and here. TODO: remove it when we have separate build for lite interpreter.
+        opss.emplace_back(c10::ivalue::Tuple::create({"_" + opname.name, opname.overload_name}));
```

Comment:
This renaming should not be part of something that hits disk. It can be done when the op code is looked up in the mobile interpreter.